Mathlib/Data/Finset/PiAntidiagonal.lean

```diff
@@ -105,13 +105,13 @@ lemma mem_finAntidiagonal₀ (d : ℕ) (n : μ) (f : Fin d →₀ μ) :
 
 end Fin
 
-section piAntidiagonal
+section finsuppAntidiag
 
 variable [DecidableEq ι]
 variable [AddCommMonoid μ] [HasAntidiagonal μ] [DecidableEq μ]
 
 /-- The Finset of functions `ι →₀ μ` with support contained in `s` and sum `n`. -/
```

Comment:
Is it perhaps worth adding a small explanation about why the code is what this comment claims it is?

Reply:
Sorry I didn't write this comment and am not sure how to update it